### PR TITLE
Switch to QMK firmware 0.19.x or above

### PR DESCRIPTION
**Pull request by koron**
Includes (done + planned): 

* https://github.com/Yowkees/keyball/pull/182
* Changes of GHA configuration
* Keyball46 care
* One47 care

Issues:
* [x] fix for Keyball46
* [x] fix for One47
* [x] stop to use `continue-on-error: true` in GHA workflow #184
* [x] bump up device versions to 1.0.0 or above.
* [x] Remap-app supports QMK firmware 0.19.x or above
* [ ] Switch QMK to 0.22.2 or abobe

This branch will be merged to main branch when Remap-app supports QMK firmware 0.19.x or above.

### CODE REVIEW

**Comment by koron** [comment]
> fix for One47

Just only moving USB info to JSON.
https://github.com/Yowkees/keyball/actions/runs/3979054959/jobs/6821222375

**Comment by koron** [comment]
> bump up device versions to 1.0.0

done by 939801c

File: qmk_firmware/keyboards/keyball/lib/keyball/keyball.c [hunk 1]
```diff
@@ -500,8 +500,8 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
         // process KC_MS_BTN1~8 by myself
         // See process_action() in quantum/action.c for details.
         case KC_MS_BTN1 ... KC_MS_BTN8: {
-            extern void register_button(bool, enum mouse_buttons);
-            register_button(record->event.pressed, MOUSE_BTN_MASK(keycode - KC_MS_BTN1));
+            extern void register_mouse(uint8_t mouse_keycode, bool pressed);
+            register_mouse(keycode, record->event.pressed);
```
Comment (AlgusDark) [inline qmk_firmware/keyboards/keyball/lib/keyball/keyball.c:504]: I believe this one is not needed anymore.
https://github.com/qmk/qmk_firmware/commit/dbe214c3268d9a316a137471a7679f7e45046e49

If I'm not wrong, then the process_action() already handles this special case.

**Comment by koron** [comment]
Remap supports QMK Firmware 0.22.2 or higher.
https://qmk018.remap-keys.app/docs/support-qmk-022

So we  need to follow it.

**Comment by koron** [comment]
I created `qmk016` branch to keep QMK 0.16 base codes.

**Comment by koron** [comment]
I have tested local compiles for below combinations:

* keyball61: test, default, via
* keyball39: via
* keyball44: via
* one47: via
* keyball46: via

**Comment by koron** [comment]
I confirmed it works with QMK 0.19.10.

Then I'll update  QMK 0.22.2 or above (maybe 0.22.3 latest)

**Pull request merged**